website/source/docs/platform/k8s/tls-on-existing-cluster.html.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: "docs"
+page_title: "Upgrading"
+sidebar_current: "docs-platform-k8s-ops-tls-on-existing-cluster"
+description: |-
+  Configuring TLS on an existing Consul cluster running in Kubernetes
+---
+
+# Configuring TLS on an Existing Cluster
+
+As of Consul Helm version `0.16.0`, the chart supports TLS for communication
+within the cluster. If you already have a Consul cluster deployed on Kubernetes,
+you may want to configure TLS in a way that minimizes downtime to your applications.
+Consul already supports rolling out TLS on an existing cluster without downtime.
+However, depending on your Kubernetes use case, your upgrade procedure may be different.
+
+## Gradual TLS Rollout without Consul Connect
+
+If you're **not using Consul Connect**, follow this process.
+
+1. Run a Helm upgrade with the following config:
+   ```yaml
```

Comment:
just noticed on the preview site,  you need a newline before all code blocks otherwise it doesn't render properly

Reply:
good catch, thank you!